InternxtDesktop/AppDelegate.swift

```diff
@@ -283,6 +283,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         } else {
             self.openWidget(delayed: true)
         }
+        ScheduledBackupManager.shared.resumeBackupScheduler()
```

Comment:
Can we recheck the backup scheduler every time the user opens the widget also? just to be sure